README.md

```diff
@@ -0,0 +1,48 @@
+# Virtual Pet
+
+A simple virtual pet program created in JavaScript.
+<br><br>
```

Comment:
You don't need to add `<br>` tags in markdown language, just press enter a few times :)